Fix Excessive Space Around Demographics Pie Chart on Mobile View

mai-repo opened the pull request:
# Description
The demographics section, specifically the pie chart, was showing excessive space around it in mobile view, causing the chart to appear oddly spaced and requiring unnecessary scrolling.

# Changes Made
- Adjusted the height of the pie chart for better mobile responsiveness.
- Modified the SASS in `static/sass/style.scss` to set the pie chart height to a fixed value on mobile screens.
  
# Files Changed:
- `static/sass/style.scss`
- `static/css/style.css.map`
- `static/css/style.css`

# Related Issue
- Issue Number: Mobile View Demographics Pie Chart Spacing #571 

Type of Change
- [x] Bug fix / Enhancement

# Objective
- Resolve the issue of excessive space around the demographics pie chart on mobile views, improving responsiveness and eliminating unnecessary scrolling.

# Acceptance Criteria:
- [x] The pie chart should display without excessive space around it on mobile devices.
- [x] The pie chart height should be adjusted appropriately, making it responsive and fitting the available screen space on mobile.

# Visuals:
**Before:**
<img width="409" alt="Image" src="https://github.com/user-attachments/assets/bc240861-7c50-440f-834f-dbe2a32c6727" />

**After:**
<img width="374" alt="Mobile-Responsive" src="https://github.com/user-attachments/assets/8b88c0ed-782f-4b0c-8903-ce2501b60f68" />

**How to test:**
1. Run the website on mobile.
2. Navigate to a page with demographics, such as "sponsor".
3. Ensure that the pie chart now fits properly with less space around it and no unnecessary scrolling.

# Reviewers:
@daaimah123, @kaylahrose, @mx-ruthie, and @monikkaelyse, please review the `static/sass/style.scss` changes where the pie chart height adjustment was made.

mx-ruthie reviewed (commented):
Wondering how I can actually view this since it's not live to the website til you merge it. 

I noticed the squished thing that Monikka mentioned and I wonder if there's not a way to reduce padding or margin from things around it to clear space rather than use the !important tag at the px level.

mai-repo commented:
@monikkaelyse and @mx-ruthie Thank you for reviewing and all of your suggestions! I have implemented the following changes to address them: 

- Edited `static/js/piechart.js` to adjust the height and width of the pie chart, as it was rendering too large.
- Added padding to ensure proper spacing around the pie chart.
- Removed the pie chart from a <div> in `templates/full-time-program.html`.
- Eliminated the **!important rule** and reduced the padding in the media query instead. 

**Note:** The spacing issue was caused by justify-content spacing, which was spacing the chart evenly based on the container's height. So, reducing margins and padding didn’t resolve the problem—we needed to reduce the size of the pie chart itself.

Here is a recorded video walkthrough of these changes: 

https://github.com/user-attachments/assets/3e3d1c8e-88e4-4a88-8023-cb9ffd931465

stmcpeters commented:
good work fixing this bug Mai! LGTM 🚀

stmcpeters reviewed (approved):
Good work fixing this bug Mai! LGTM 🚀

daaimah123 reviewed (approved):
Approved feel free to merge

mai-repo commented:
Got permission to merge and merging into `develop branch`
